CMSgov/design-system
# [WNMGDS-1576] USABanner component theme

## Summary

- Adds `USABanner` component theme
- Colors were off-brand and hard-coded previously, updated to use brand colors
- Updated loki references

## How to test

Can look at screenshots to see updates to references, the references aren't expanded in storybook automatically, can `yarn install && yarn build && yarn run` to compare to https://design.cms.gov/
